tests/functional/spec/features/dynamic_answers_list_value_source.spec.js

```diff
@@ -0,0 +1,192 @@
+import DriverPage from "../../generated_pages/dynamic_answers_list_source/any-supermarket.page";
+import DynamicAnswerPage from "../../generated_pages/dynamic_answers_list_source/dynamic-answer.page";
+import DynamicAnswerOnlyPage from "../../generated_pages/dynamic_answers_list_source/dynamic-answer-only.page";
+import ListCollectorPage from "../../generated_pages/dynamic_answers_list_source/list-collector.page";
+import ListCollectorAddPage from "../../generated_pages/dynamic_answers_list_source/list-collector-add.page";
+import ListCollectorRemovePage from "../../generated_pages/dynamic_answers_list_source/list-collector-remove.page";
+import SetMinimumPage from "../../generated_pages/dynamic_answers_list_source/minimum-spending.page";
+import SectionSummaryPage from "../../generated_pages/dynamic_answers_list_source/section-summary.page";
+
+describe("Dynamic answers list value source", () => {
+  const labels = 'label[class="ons-label"]';
+  const inputs = 'input[class="ons-input ons-input--text ons-input-type__input ons-input-number--w-3"]';
+  const group = 'div[id="group-2"]';
+  const summaryTitles = 'dt[class="ons-summary__item-title"]';
+  const summaryValues = 'dd[class="ons-summary__values"]';
+  const summaryActions = 'dd[class="ons-summary__actions"]';
```

Comment:
This is not ideal and prone to error but being pragmatic so we can get this in. We can revisit generate pages/selectors as a whole when we have a bit more time.